### ISSUE TITLE
Where is the `Iso' Text String`?

### ISSUE THREAD

**Issue by kindaro** [ekmett/lens#1062]
Take a look at this definition:

```Haskell
string ∷ Iso' Text String
string = iso Text.unpack Text.pack
```

I thought I shall find it in `lens`. I do not see why it should not be there:

* `lens` already depends on `text`.
* There is already a type class `Reversing` that defines an [instance](https://hackage.haskell.org/package/lens-5.2.3/docs/Control-Lens-Iso.html#t:Reversing) for `Text`.

Yet I could not find anything like this.

* If it is not there, can we add it?
* If it is there, can we make it easier to find?

**Comment by ocharles** [comment]
It's `packed`: https://hackage.haskell.org/package/lens-5.2.3/docs/Data-Text-Lens.html

**Comment by kindaro** [comment]
Makes sense. I did not think this stuff would be in a module of its own.

However, Hoogle cannot find these functions by type. I go by the link <https://hoogle.haskell.org/?hoogle=Iso%27%20String%20text> and there is nothing. Hoogle can find them by name though: the link <https://hoogle.haskell.org/?hoogle=packed> gives all the right hits from `lens`. What am I doing wrong?

**Comment by ocharles** [comment]
I don't think Stackage knows about type aliases

**Comment by RyanGlScott** [comment]
Hoogle is a bit quirky. If you search for [`:: Iso' String text`](https://hoogle.haskell.org/?hoogle=%3A%3A%20Iso%27%20String%20text) (note the double colons), then `packed` _does_ appear in the search results, albeit not as the first result. If you refine the search further to [`:: Iso' String Text`](https://hoogle.haskell.org/?hoogle=%3A%3A%20Iso%27%20String%20Text), then it's the first result.

**Comment by phadej** [comment]
... it's indeed quirky, as the expansion `:: Iso String String Text Text` doesn't result in `packed`. I.e. you need to know what to search for find anything. Hoogle treats `type`s as opaque, and in type-alias heavy library it's not great (even with `optics` where full expansion still "makes sense", Hoogle doesn't find anything).
